packages/cli/src/lib/build-strategies/strategies/DockerImageStrategy.ts

```diff
@@ -190,37 +194,37 @@ export class ImageBuildStrategy extends BuildStrategy<BuildImageId> {
 
       await runCommand(
         `docker cp root-${imageName}:/project/build/${buildArtifact} ${outputDir}`,
-        this.project.quiet
+        this.project.logger
       );
 
-      await runCommand(`docker rm -f root-${imageName}`, this.project.quiet);
+      await runCommand(`docker rm -f root-${imageName}`, this.project.logger);
 
       if (useBuildx) {
         if (removeBuilder) {
-          await runCommand(`docker buildx rm ${imageName}`, this.project.quiet);
+          await runCommand(
+            `docker buildx rm ${imageName}`,
+            this.project.logger
+          );
         }
       }
       if (removeImage) {
-        await runCommand(`docker rmi ${imageName}`, this.project.quiet);
+        await runCommand(`docker rmi ${imageName}`, this.project.logger);
       }
     };
 
-    if (this.project.quiet) {
-      return await run();
-    } else {
-      const args = {
-        path: displayPath(outputDir),
-        image: imageName,
-      };
-      return (await withSpinner(
-        intlMsg.lib_helpers_docker_copyText(args),
-        intlMsg.lib_helpers_docker_copyError(args),
-        intlMsg.lib_helpers_docker_copyWarning(args),
-        async (_spinner) => {
-          return await run();
-        }
-      )) as void;
-    }
+    const args = {
+      path: displayPath(outputDir),
+      image: imageName,
+    };
+    return await logActivity<void>(
+      this.project.logger,
+      intlMsg.lib_helpers_docker_copyText(args),
+      intlMsg.lib_helpers_docker_copyError(args),
+      intlMsg.lib_helpers_docker_copyWarning(args),
+      async (_spinner) => {
```

Comment:
Probably want to rename this to `_logger` just for the sake of clarity? 😄 There's a few of these within the PR, `ctrl+F spinner`
Or just remove it as you've done in other places.